src/styles/global.css

```
@@ -209,6 +209,188 @@ body[data-mobile-nav-open] {
+	outline-offset: 3px;
+	text-decoration: none;
+}
+
+.post-body {
+	font-size: clamp(1rem, 0.97rem + 0.28vw, 1.125rem);
+	line-height: 1.8;
+	color: var(--text-main);
+	width: 100%;
+	min-width: 0;
+	overflow-wrap: break-word;
+	word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)